### DISCUSSION TITLE
Feature Request: Contact

### DISCUSSION THREAD

**Discussion by jvanderaa** [nautobot/nautobot#1071] [Ideas]
### Environment
* Python version: 3.9
* Nautobot version: 1.1.4

### Proposed Functionality

As part of the Nautobot Plugin Device Lifecycle Management plugin the model of a contact was introduced there. 

### Use Case

1. This is something that could be used in many other places as well. Such as assigning ownership of devices.
2. Ownership of sites
3. By moving into core then other plugins can leverage the model and have a single unified contact experience

### Database Changes

Create new data model, user

### External Dependencies

None

**Comment by jathanism** [comment]
How about we turn this into a discussion to cover how we might model this and how it would be associated with other objects?

**Comment by glennmatthews** [comment]
Related previous issues:
https://github.com/nautobot/nautobot/issues/8
https://github.com/nautobot/nautobot/issues/230

**Reply by jvanderaa** [reply]
I didn't see those when I started the thread. I think I searched for contact on the menu bar. Those have the data that would likely cover it.

**Comment by jvanderaa** [comment]
@glennmatthews @bryanculver @jathanism @lampwins 

Any updates on where this may be at?

**Comment by jvanderaa** [comment]
One of the concepts would be around being able to have a product owner assigned devices. So we would need to have a `ContactType` field.

**Comment by bryanculver** [comment]
This discussion has been moved to #230.
